### PR TITLE
Add tarantool datastore

### PR DESCRIPTION
**Pull request by awekening2**
Added support for Tarantool

### CODE REVIEW

File: mcs/resource_mcs_db_cluster.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func resourceDatabaseCluster() *schema.Resource {
 							ForceNew: true,
 							ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
 								v := val.(string)
-								if v != Galera && v != Postgres {
+								if v != Galera && v != Postgres && v != Tarantool {
```
Comment (irlndts) [inline mcs/resource_mcs_db_cluster.go:143]: i guess construction like
```
switch v {
    case Galera, Postgres, Tarantool: // do nothing
    default: errs = .... // catch error 
}
```
would be more pleasant for an eye

Comment (pashapdev) [inline mcs/resource_mcs_db_cluster.go:143]: maybe use something like this:

`
if availableClusterDatastores(v){


func availableClusterDatastores(ds string) bool {
	for _, clusterDatastore := range getClusterDatastores() {
		if ds == clusterDatastore {
			return true
		}
	}
	return false
}`

Comment (pashapdev) [reply mcs/resource_mcs_db_cluster.go:143]: so there will be no need to change the code in 2 places

**Comment by irlndts** [comment]
can you add a line to changelog.md pls under new tag please

File: mcs/resource_mcs_db_cluster.go (hunk 1 above)
Comment (schirevko) [reply mcs/resource_mcs_db_cluster.go:143]: I think we can refactor this whole func to use terraform provided validation.StringInSlice: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/validation#StringInSlice

`
ValidateFunc: validation.StringInSlice(getClusterDatastores(), true)
`

Comment (irlndts) [reply mcs/resource_mcs_db_cluster.go:143]: an option, agree

File: docs/changelog.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ description: |-
 
 # MCS Provider's changelog
 
+#### v0.6.8
```
Comment (irlndts) [inline docs/changelog.md:10]: do rebase please, v0.6.8 has been already realesed and add a line between versions

**Review by irlndts — COMMENTED** [review]
what about moving this to a function isOperationSupported(type) error
```
		switch instanceResource.DataStore.Type {
		case Redis, Tarantool:
			return fmt.Errorf("operation not supported for this datastore")
		}
```

**Review by irlndts — APPROVED** [review]

**Pull request merged**